pkgs/development/python-modules/clingo/default.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  python3,
+  python3Packages,
+  clingo,
+}:
+
+python3Packages.buildPythonPackage rec {
+  inherit (clingo) pname version src;
+
+  postPatch = ''
+    cd libpyclingo
+    python compile.py h
+  '';
+
+  pyproject = true;
+  build-system = [ python3Packages.setuptools ];
+
+  dependencies = with python3Packages; [ cffi ];
+  nativeBuildInputs = [ python3 ];
+  buildInputs = [ clingo ];
+
+  postInstall = ''
+    mkdir -p "$out/${python3.sitePackages}/clingo/lib/"
+    cp -Tr "${clingo}/lib" "$out/${python3.sitePackages}/clingo/lib/"
```

And reference the generic `python` attribute instead.

```suggestion
    mkdir -p "$out/${python.sitePackages}/clingo/lib/"
    cp -Tr "${clingo}/lib" "$out/${python.sitePackages}/clingo/lib/"
```